### ISSUE TITLE
[GraphQL] Only the first asset field inside a repeater field gets saved

### ISSUE THREAD

**Issue by hw-01010** [verbb/formie#1832]
### Describe the bug

I currently have a problem, saving assets inside a repeater field via GraphQL mutation.
The first file is saved normally, but the other files don't save.

![image](https://github.com/verbb/formie/assets/112609658/0aae6976-8fc2-4e23-bbd3-27d4d6190b12)

I use the following GraphQL Mutation:
```graphql
mutation MyMutation($repeaterField: testststststst_asdfasdf_FormieRepeaterInput) {
  save_testststststst_Submission(asdfasdf: $repeaterField) {
    id
  }
}
```

Query Variable:
```json
{
  "repeaterField": {
    "rows": [
      {
        "asdfasdf": [
          {
            "fileData": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNgYAAAAAMAASsJTYQAAAAASUVORK5CYII="
          }
        ]
      },
      {
        "asdfasdf": [
          {
            "fileData": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNgYAAAAAMAASsJTYQAAAAASUVORK5CYII="
          }
        ]
      }
    ]
  }
}
```

I have already tried to use different Filenames, different files, creating a new From, nothing worked, so I think this is a Bug.

### Steps to reproduce

1. Create a From with a repeater field and a File Upload field
2. Execute provided GraphQL Mutation
3. Check Form Submisson

### Form settings

- Multi-page form: No
- Submission Method: GraphQL
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.8.9

### Plugin version

2.1.11

### Multi-site?

yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.11"`.

**Comment by engram-design** [comment]
Fixed in [2.1.12](https://github.com/verbb/formie/releases/tag/2.1.12)
